### PR TITLE
[compiler-rt] Remove a few workarounds for FreeBSD 9.x

### PR DESCRIPTION
**Pull request by brad0**
Support for FreeBSD 11.x was dropped so garbage collect a few FreeBSD
9.x workarounds and make 12.x the oldest supported releases.

### CODE REVIEW

**Comment by brad0** [comment]
Ping.

File: compiler-rt/lib/sanitizer_common/sanitizer_freebsd.h [hunk 1]
```diff
@@ -1,137 +0,0 @@
-//===-- sanitizer_freebsd.h -------------------------------------*- C++ -*-===//
```
Comment (emaste) [inline compiler-rt/lib/sanitizer_common/sanitizer_freebsd.h:1]: why is this file no longer needed

Comment (brad0) [reply compiler-rt/lib/sanitizer_common/sanitizer_freebsd.h:1]: https://github.com/llvm/llvm-project/commit/1f386f0f364183d85d636ff7be160712fc4bb5b8

The header was added as part of a workaround for 9.x and after that is removed it is basically an empty header. Just...
```
#if SANITIZER_FREEBSD

#endif
```

**Review by MaskRay — APPROVED** [review]

**Review by DimitryAndric — APPROVED** [review]
Yes, this is a nice cleanup.

**Pull request merged**